src/Contao/View/Contao2BackendView/Subscriber/WidgetBuilder.php

```diff
@@ -472,6 +472,7 @@ private function prepareWidgetAttributes(ModelInterface $model, PropertyInterfac
         $value = $this->valueToWidget($model, $property);
 
         $propExtra['required'] = ('' === $value) && !empty($propExtra['mandatory']);
+        $this->setPropExtraDisabled($property, $propExtra);
```

Comment:
You return a propExtra in the method. After call the method, is the manipulation not available.

Reply:
👍